### PR TITLE
Syndicate business card

### PR DESCRIPTION
**Pull request by pofitlo-Git**
## About the PR

A new uplink item is added to the game - the syndicate's business card. in general, it is quite useless and is only needed for role play. You can write a message on the back

## Technical details

it costs nothing, but it is limited in purchase. you can only buy 3 of them

## Media

image
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

- add: Added a syndicate business card to the uplink.

### CODE REVIEW

File: Resources/Locale/en-US/paper/syndicate-business-card.ftl
Comment (lzk228) [inline Resources/Locale/en-US/paper/syndicate-business-card.ftl:2]:
```suggestion
syndicate-business-card-base = It's nothing personal, it's just business

```

File: Resources/Locale/en-US/store/uplink-catalog.ftl
Comment (lzk228) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:446]: all words start with uppercase, that's called title case in english

File: Resources/Locale/en-US/store/uplink-catalog.ftl
Comment (lzk228) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:447]: first letter uppercase and dot at the end

File: Resources/Prototypes/Entities/Objects/Misc/business card.yml
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Misc/business card.yml:1]: wouldn't be better to put it in paper.yml and set BasePaper as parent?

File: Resources/Textures/Objects/Misc/Business_сard.rsi/Synicate card.png
Comment (lzk228) [inline Resources/Textures/Objects/Misc/Business_сard.rsi/Synicate card.png:1]: name should be `syndicate_card.png`

**Review by lzk228 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Misc/business card.yml
Comment (pofitlo-Git) [reply Resources/Prototypes/Entities/Objects/Misc/business card.yml:1]: I saw an offer to make more of these cards. for a bar, for example, or for something else, idk. so maybe it would be better to allocate it to a separate file

**Comment by UbaserB** [comment]
This is so based

File: Resources/Prototypes/Entities/Objects/Misc/business card.yml
Comment (Ertanic) [inline Resources/Prototypes/Entities/Objects/Misc/business card.yml:9]:
```suggestion
      state: Synicate_card
```
You forgot to fix the name of the state in the prototype.
image

**Comment by pofitlo-Git** [comment]
image
logo

maybe someone someday draw a normal logo....

File: Resources/Textures/Interface/Paper/paper_heading_syndicate_logo.svg
Comment (eoineoineoin) [inline Resources/Textures/Interface/Paper/paper_heading_syndicate_logo.svg:1]: Looks like this file wasn't saved correctly? It's just a resized version of paper_heading_virus.svg?

File: Resources/Prototypes/Entities/Objects/Misc/paper.yml
Comment (eoineoineoin) [inline Resources/Prototypes/Entities/Objects/Misc/paper.yml:40]: Is this change intentional? This is just setting to the default value, so can be removed - the change also makes the comment incorrect.

File: Resources/Textures/Objects/Misc/Business_card.rsi/meta.json
Comment (eoineoineoin) [inline Resources/Textures/Objects/Misc/Business_card.rsi/meta.json:11]: Typo. Also in the "Synicate_card.png" filename.

File: Resources/Textures/Interface/Paper/paper_background_black.svg
Comment (eoineoineoin) [inline Resources/Textures/Interface/Paper/paper_background_black.svg:1]: This file does not seem to be the same one which generated paper_background_black.svg.96dpi.png?

**Comment by HenhenIII** [comment]
Based

File: Resources/Locale/en-US/store/uplink-catalog.ftl
Comment (slarticodefast) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:447]:
```suggestion
uplink-business-card-name = Syndicate business card.
uplink-business-card-desc = A business card that you can give to someone to demonstrate your involvement in the syndicate or leave at the crime scene in order to make fun of the detective. You can buy no more than three of them.
```

File: Resources/Prototypes/Catalog/uplink_catalog.yml
Comment (slarticodefast) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1775]:
```suggestion
  name: uplink-business-card-name
  description: uplink-business-card-desc
```

File: Resources/Prototypes/Entities/Objects/Misc/business card.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Misc/business card.yml:1]: Rename the file to business_card.yml

File: Resources/Prototypes/Entities/Objects/Misc/business card.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Misc/business card.yml:9]:
```suggestion
      state: sydicate_card
```

File: Resources/Textures/Objects/Misc/Business_card.rsi/meta.json
Comment (slarticodefast) [inline Resources/Textures/Objects/Misc/Business_card.rsi/meta.json:11]:
```suggestion
            "name": "syndicate_card"
```
Same for the file name.
The folder should be renamed to business_card.rsi

File: Resources/Prototypes/Entities/Objects/Misc/business card.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Misc/business card.yml:4]: This should just inherit from the Paper entity in paper.yml and only overwrite what needs to be changed.

**Review by slarticodefast — CHANGES_REQUESTED** [review]

**Comment by slarticodefast** [comment]
The default isn't centered at the moment like in the PR image. You can add some whitespace to the ftl string using
`{"           "}`
Also please add an changelog to your PR description to notify players of your addition.

**Comment by slarticodefast** [comment]
Also a request from one of our art directors for the item sprite:
"make sure the item isnt in 3/4ths, is really all i have to say. goes against the artstyle guidelines im pretty sure.
the sprite could maybe use some rounded edges to make it more card-like, too"

**Comment by slarticodefast** [comment]
Test fail is real.
Also I will ask the art directors for approval of the new sprite.

**Comment by pofitlo-Git** [comment]
> Test fail is real. Also I will ask the art directors for approval of the new sprite.

it's because any "paper" sould have "write" state with sprite. And I don't know exactly how resolve this

**Comment by pofitlo-Git** [comment]
@slarticodefast I think that it can work now

**Comment by slarticodefast** [comment]
Awesome, I'll give it another in-game test later.
The item sprite might need some more adjustments according to the art directors:
- the sprite should not be in 3/4ths perspective
- "the bottom highlight is what's making it look 3/4ths"
- "the shading on the s is kinda messy"
All I got as a review from them so far, I can ask for more clarification if needed.

**Comment by Ilya246** [comment]
maybe you should be able to fax them

**Comment by eoineoineoin** [comment]
These comments are marked as resolved, but are still outstanding:

https://github.com/space-wizards/space-station-14/pull/32162#discussion_r1760060393
https://github.com/space-wizards/space-station-14/pull/32162#discussion_r1759782733

You've also modified the whitesepace on every single line in Resources/Textures/Objects/Misc/bureaucracy.rsi/meta.json - can you fix that?

**Comment by pofitlo-Git** [comment]
> These comments are marked as resolved, but are still outstanding:
> 
> [#32162 (comment)](https://github.com/space-wizards/space-station-14/pull/32162#discussion_r1760060393) [#32162 (comment)](https://github.com/space-wizards/space-station-14/pull/32162#discussion_r1759782733)
> 
> You've also modified the whitesepace on every single line in Resources/Textures/Objects/Misc/bureaucracy.rsi/meta.json - can you fix that?

yes It's my mistake but I don't understate what do u mean about modified the whitesepace on every single line. It's look same...

**Comment by beck-thompson** [comment]
Also I could be wrong but I still think there is a merge conflict?

**Comment by slarticodefast** [comment]
> Also I could be wrong but I still think there is a merge conflict?

A direct merge conflict should get shown in the github actions.
There can be silent merge conflicts if there are namespace or API changes (that's why we have the breaking changes section).
I usually merge master before doing ingame tests to make sure

**Comment by cohanna** [comment]
> yes It's my mistake but I don't understand what do u mean about modified the whitesepace on every single line. It's look same...
Resources/Textures/Objects/Misc/bureaucracy.rsi/meta.json
most notably line 10 has a larger indent (which propagates to the lines below)
but just about every line has had at least one extra indent

you can view the diff if you go to the "files changed" tab of this pr.

**Comment by pofitlo-Git** [comment]
okey now It's done

**Comment by eoineoineoin** [comment]
@pofitlo-Git I appreciate this has had a lot of back and forth to fix issues - thanks for your patience here, it gets easier, really!

@slarticodefast I'm not sure that pofitlo knows how to resolve merge conflicts, so in the interests of saving everyone a bit more hassle, I've resolved the merge conflict and fixed another typo in https://github.com/eoineoineoin/space-station-14/tree/pofitlo/businesscard -  could you fast-forward their branch to that?

**Comment by slarticodefast** [comment]
> I'm not sure that pofitlo knows how to resolve merge conflicts, so in the interests of saving everyone a bit more hassle, I've resolved the merge conflict and fixed another typo in https://github.com/eoineoineoin/space-station-14/tree/pofitlo/businesscard - could you fast-forward their branch to that?

Sure, thanks for helping out. I'll give the business card a final code review and in-game test as soon as possible and this should be good to merge then.

**Comment by pofitlo-Git** [comment]
> @pofitlo-Git I appreciate this has had a lot of back and forth to fix issues - thanks for your patience here, it gets easier, really!
> 
> @slarticodefast I'm not sure that pofitlo knows how to resolve merge conflicts, so in the interests of saving everyone a bit more hassle, I've resolved the merge conflict and fixed another typo in https://github.com/eoineoineoin/space-station-14/tree/pofitlo/businesscard - could you fast-forward their branch to that?

thank you very mach! I really awful for git

File: Resources/Prototypes/Entities/Objects/Misc/business_card.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Misc/business_card.yml:3]:
```suggestion
  name: syndicate business card
```

**Review by slarticodefast — APPROVED** [review]

**Comment by slarticodefast** [comment]
Thank you for your contribution and your first PR :)

**Pull request merged**
